packages/clangml/clangml.4.5.0/opam

```diff
@@ -23,7 +23,7 @@ depends: [
   "metaquot" {>= "0.4.0"}
   "refl" {>= "0.4.0"}
   "odoc" {with-doc & >= "1.5.1"}
-  "ppxlib" {>= "0.23"}
+  "ppxlib" {>= "0.23" & < "0.26.0"}
```

Comment:
This is actually a metapp failure for now. I think we should remove this upper bound for now